src/views/List.jsx

```diff
@@ -1,23 +1,41 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	/* TO DO: Make separate resuable input component with a filter feature*/
+
+	const filterList = data.filter((item) => {
+		if (searchTerm === '') {
```

Comment:
Curious: what happens if a user enters multiple empty spaces in the input? This is definitely an edge case, but something to consider.